src/java/org/apache/cassandra/index/sai/plan/Plan.java

```diff
@@ -1951,11 +1951,13 @@ public static double hrs(double raw)
     }
 
     /**
-     * Describes the expected distribution of data access patterns for a plan node.
+     * Describes the expected data access patterns for a plan node.
      * <br>
-     * Each access pattern is represented by a pair of values:
-     * a count (number of expected occurrences) and a distance (skip distance).
-     * For performance, these are split into arrays of primitives.
+     * Each access pattern is assumed to follow uniform distribution and
+     * is represented by a pair of values:
+     * - a count (number of expected occurrences) and
+     * - a distance to next occurrence (skip distance).
```

Comment:
nit: well, if we earlier say the accesses are uniformly distributed (in terms of which rows are picked), then maybe we should add here that this is a statistically expected (aka average) distance across the whole set of accesses. Because of course, skip distances are not const.